RPIPs/RPIP-46.md

```diff
@@ -1,103 +0,0 @@
-   4. The pDAO MAY change `increase_no_share_seal_count` via vote
-9. The initial settings SHALL be:
-   1. `node_operator_commission_share`: 3.5%
-   2. `voter_share`: 5%
-   3. `surplus_share`: 5.5%
-   4. `increase_no_share_seal_increment`: 0.5%
-   5. `increase_no_share_seal_count`: 6
-   6. `allowlisted_controllers`: []
-10. Prior to the release of Saturn 1, a ranked-choice vote MUST be held to select a mechanism for surplus revenue share
-    1. The choices MAY include [RPIP-45: RPL Burn](RPIP-45.md), [RPIP-50: RPL LP](RPIP-50.md), or using higher `voter_share`
-    2. If higher `voter_share` is selected, the entirety of the ["Specification taking effect with Saturn 2"](#specification-taking-effect-with-saturn-2) section below SHALL be deleted
-    3. The selected mechanism MAY be implemented in Saturn 1
-       1. If not, the revenue SHALL be held in reserve until the mechanism is implemented. In such a case, once the mechanism is implemented, the revenue SHALL be distributed in the same amount of time it took to build up, or faster.
-    4. The selected mechanism MUST be implemented in Saturn 2
-    5. When implemented, `surplus_share` SHOULD be renamed to something more specific
-
-## Specification taking effect with Saturn 2
-1. Updating `voter_share`:
-   1. A new function SHALL be available to update `voter_share`, which MAY be called by anyone
-   2. It MUST revert if it's been called within the last 45 days
-   3. It MUST revert if the total RPL eligible to vote is between `voter_share_target_min` and `voter_share_target_max` (inclusive)
-   4. If <`voter_share_target_min` of total RPL is eligible to vote and the function succeeds:
-      1. `voter_share` is increased to `voter_share * (1+voter_share_relative_step)`
-      2. `surplus_share` is decreased by the difference between the old and new `voter_share`
-         1. If this would reduce `surplus_share` below 0%, the function call MUST revert 
-   5. If >`voter_share_target_max` of total RPL is eligible to vote and the function succeeds:
-      1. `voter_share` is decreased to `voter_share / (1+voter_share_relative_step)`
-      2. `surplus_share` is increased by the difference between the old and new `voter_share`
-   6. Because this involves _voters_ modifying `voter_share`, there is an acknowledged conflict of interest here. As a result, changing this method of "Updating `voter_share`" SHALL require a supermajority vote with at least 75% of the vote in support of any change.
-2. `voter_share_relative_step`, `voter_share_target_min`, and `voter_share_target_max` MAY be updated by pDAO vote; however, it SHALL require a supermajority vote with at least 75% of the vote in support of any change.
-3. The initial settings SHALL be:
-   1. `voter_share_relative_step`: 15%
-   2. `voter_share_target_min`: 55%
-   3. `voter_share_target_max`: 65%
```

Comment:
A priori, it isn't clear that the step size is fine enough and the range between min and max is wide enough to allow for a steady state. Meaning, we could end up in a scenario where we increase one step and end up above the maximum, we increase one step and end up below the minimum and then repeat this process indefinitely. 

Might be possible to do some modeling on this, but doesn't seem like a huge priority. Can always address it with settings changes.